### PR TITLE
PUBSTWO-1697

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

**Review by orosz-usgs — COMMENTED** [review]
Looks like there might be an extra character on lines 11 and 15???
I am seeing a small darker green cursor sized box displayed in the diff (before the -)
might be fine.

**Review by orosz-usgs — COMMENTED** [review]
for the Readme.md, how do you add the insecure profile to a build?

**Review by orosz-usgs — APPROVED** [review]

**Review by mbucknell — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/usgs/pubs-services/compare/pubs-services-1.0.0...master)
+### Added
+  - Ability to bypass security for local development
 ### Changed
-- Source of IPDS data for dissemination step.
+  - Source of IPDS data for dissemination step.
```
Comment (dsteinich) [inline CHANGELOG.md:11]: The two spaces here should fix the codacy issue - https://app.codacy.com/manual/usgs_wma_dev/pubs-services/issues?bid=13642041&filters=W3siaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6WyJDb2RlU3R5bGUiXX1d

File: README.md [hunk 2]
```diff
@@ -60,7 +60,11 @@ spring.security.user.password: changeMe
 logging.gelf.host: "tcp:localhost"
 logging.gelf.port: 12201
 ```
+Optionally adding the following will disable security for local development.
```
Comment (dsteinich) [inline README.md:63]: These are run time parameters, not build parameters.

**Pull request merged**